lib/diagnostic.rb

```diff
@@ -6,6 +6,10 @@ class Diagnostic
   # method that returns a Fixnum.
   def longest(enum)
     # Your code goes here
+    longest = enum.reduce do |memo, obj|
+      memo.length > obj.length ? memo : obj
+    end
+    p longest
```

Comment:
Return, not print. In fact, `print` returns `undefined`, so this method will return `undefined`.